posts/2020-02-12-using-dotty-union-types-with-akka-typed-part-II.adoc

```diff
@@ -0,0 +1,285 @@
+= Using Dotty Union types with Akka Typed - Part II
+eloots
+v1.0, 2020-02-19
+:title: Using Dotty Union types with Akka Typed - Part II
+:tags: [dotty, akka]
+
+== Introduction
+
+After posting the first article on https://www.lunatech.com/2020/02/using-dotty-union-types-with-akka-typed[Using Dotty Union types with Akka Typed], I received https://twitter.com/hseeberger/status/1227660145125199877?s=20[valuable feedback] via Twitter which made me decide to write this follow-up article.
+
+I will clean-up a few loose ends and add some new insights. Let's start by listing them:
+
+1. In the Scala 2 version of the code, a message adapter is used, but the `WrappedPongResponse` message that wraps the response is part of the `Pinger` actor's protocol.
```

Comment:
I think I'd be more explicit here. It's not fully clear exactly what the issue is. For example you could say "is part of the `Pinger` actor's protocol _instead of xxx_." Not everyone will remember the exact protocols from the last one, so the more explicit the better.

Reply:
Adding a drawing to assist in explaining the issue